base_tier_validation/models/tier_validation.py

```diff
@@ -42,19 +42,14 @@ class TierValidation(models.AbstractModel):
         domain=lambda self: [("model", "=", self._name)],
         auto_join=True,
     )
-    to_validate_message = fields.Html(compute="_compute_validated_rejected")
-    # TODO: Delete in v17 in favor of validation_status field
```

Comment:
The fields are still superfluous (because you can now search for `validation_status == 'validated'`, and now they even take up space in the table. In https://github.com/OCA/server-ux/pull/1101 the fields are still queued for removal, but in the mean time their search method was much improved by using the stored validation_status field.

Reply:
Sorry, do you mean the `to_validate_message` field (not store) or the `validated` and `rejected` fields? 
IMO defining the `validated` and `rejected` fields as store is not a “_problem_” because table space is insignificant, and besides, they are very easy to define their value without the need to maintain _search code for it, don't you think (the idea is to improve and simplify as much as possible, and this is one of those “_easy_” things to do)?

Reply:
Given that customizations of the flow now depend on `_compute_validated` and `_compute_rejected` as per the other comment, I agree!